backend/inspirehep/records/indexer/cli.py

```diff
@@ -239,3 +240,56 @@ def _finished_tasks_count():
         )
         for failure in failures:
             LOGGER.warning(failure)
+
+
+@index.command(
+    "remap",
+    help="Remaps specified indexes. Removes all data from index during this process.",
+)
+@click.option("--yes-i-know", is_flag=True)
+@click.option(
+    "--index",
+    "-i",
+    multiple=True,
+    default=None,
+    help="Specify indexes which you want to remap (ignore prefix and postfix)",
+)
+@click.option(
+    "--ignore-checks",
+    is_flag=True,
+    help="Do not check if old index was deleted. WARNING: this may lead to one alias pointing to many indexes!",
+)
+@with_appcontext
+@click.pass_context
+def remap_indexes(ctx, yes_i_know, index, ignore_checks):
```

Comment:
```s/index/indexes```

Reply:
It's not recommended to call this option indexes as it allows to put only one index at a time. You can put more by putting it multiple times. 
In our code we use singular for `multiple` (like `pid` in indexing CLI) click also suggest to not use plural for it.